### ISSUE TITLE
ValidVariableName doesn't test a private member when PrivateNoUnderscore is excluded

### ISSUE THREAD

**Issue by mauriciofauth** [squizlabs/PHP_CodeSniffer#1880]
Private members are only tested when the name contains a leading underscore.
```xml
<rule ref="Squiz.NamingConventions.ValidVariableName">
    <exclude name="Squiz.NamingConventions.ValidVariableName.PrivateNoUnderscore" />
</rule>
```
```php
class Test
{
    private $private_member;
}
```

**Comment by jrfnl** [comment]
PR #1883 should fix this.
